modules/kubernetes/prometheus/main.tf

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Maybe we should read the client cert and key from KeyVault inside of this module?